watertap/unit_models/anaerobic_digestor.py

```diff
@@ -462,6 +462,19 @@ def build(self):
             mutable=True,
             doc="Gas-liquid transfer coefficient",
         )
+        self.electricity = Var(
+            self.flowsheet().time,
+            units=pyunits.kW,
```

Comment:
This is technically electric _power_ rather than energy if using units kW. I know we weren't strict about this in the past, but figured I'd mention now.

Reply:
I'd personally prefer calling this `power_consumption_electric`. Added the electric qualifier in case there is any thermal input that we'd consider later

Reply:
The other thing to consider is that ADs produce gas that can in turn be used to produce electricity, and I'd rather now confuse the user about whether this is electricity production or consumption.

Reply:
In the Maravalia's data, they provide an electricity cost with respect to AD and at least for this value I added, I believe it is electricity consumption

Reply:
Sure- I just meant we should rename from `electricity` to something more specific like `power_consumption_electric` or `electricity_consumption`.

Reply:
> Sure- I just meant we should rename from `electricity` to something more specific like `power_consumption_electric` or `electricity_consumption`.

I change it to `electricity_consumption`